### PR TITLE
Scorecard

### PR DESCRIPTION
**Pull request by dbrinegar**
This is OLM-ifying an operator-sdk project such that operator-sdk scorecard and operator-courier both give good scores.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,276 @@
...
 ```
 
-## demo application
+## Demo application
 
-https://github.com/dbrinegar/akka-java-cluster-openshift
+[Akka Cluster visualizer](https://github.com/dbrinegar/akka-java-cluster-openshift)
 
-## hacking
+## Hacking
```
Comment (edwardcallahan) [inline README.md]: How does one build the operator image?

File: README.md [hunk 2]
```diff
@@ -1,54 +1,276 @@
...
+## Install the controller
 
 in each namespace where akkacluster apps are desired
 
 ```bash
 kubectl apply -f ./deploy
 ```
 
-## demo application
+## Demo application
```
Comment (edwardcallahan) [inline README.md]: Can we expand this section to 'using in an Application'? No need to repeat akka docs. Maybe not much more than "set your app up to use the management lib but stop before you app.conf " is needed here?

File: hacking.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+`./deploy/*.yaml` is the operator Deployment, ServiceAccount, Role, and RoleBinding. These
+were all generated by operator-sdk, meaning nothing special here, just generic operator
+things.
+
+`./deploy/crds/app_v1alpha1_akkacluster_crd.yaml` has the custom resource definition. This is where
+new top level fields and basic validation go, if you want `kubectl` to know a valid from
+invalid AkkaCluster.
+
+`./deploy/old-catalog/` has a nested OLM package, which should be updated on releases. The
+primary source here are the "*clusterserviceversion.yaml" files, one for each version published.
```
Comment (edwardcallahan) [inline hacking.md:73]: `old-catalog`, snicker `olm`

**Review by edwardcallahan — APPROVED** [review]
Nice work, David. Thanks for taking the time to sort thru and document this.

File: README.md (hunk 1 above)
Comment (dbrinegar) [reply README.md]:
```
operator-sdk build <image>
```

but I'll add a note

File: README.md (hunk 2 above)
Comment (dbrinegar) [reply README.md]: The latest version has `Application requirements`.. does that do what you were thinking?

File: hacking.md (hunk 3 above)
Comment (dbrinegar) [reply hacking.md:73]: fixed :-)

File: README.md (hunk 2 above)
Comment (edwardcallahan) [reply README.md]: Yes, thank you.

**Pull request merged**